ui/app/serializers/service-fragment.js

```diff
@@ -3,9 +3,5 @@ import classic from 'ember-classic-decorator';
 
 @classic
 export default class ServiceFragmentSerializer extends ApplicationSerializer {
-  attrs = {
```

Comment:
Side-effect; we're no longer worrying about the connect flag on service fragments.